rotkehlchen/globaldb/handler.py

```diff
@@ -1011,6 +1011,123 @@ def add_historical_prices(entries: List['HistoricalPrice']) -> None:
 
         connection.commit()
 
+    @staticmethod
+    def add_single_historical_price(entry: HistoricalPrice) -> None:
+        """Adds the given historical price entries in the DB
```

Comment:
This is only for manual price, no? If so then please change the name of the function.

Reply:
This can store any historical price, doesn't have to be manual